### ISSUE TITLE
Ask the user the amount of distributions to display as a result

### ISSUE THREAD

**Issue by xahodo** [distrochooser/distrochooser#230]
This basically would be an option to the right/left/bottom/top of the list of distributions shown. The default would be the current amount, but it could for example be a slider, which would add or subtract distributions depending on the amount you move it to. Another option could be 3 to 5 buttons to select different amounts of distributions shown.

**Comment by cmllr** [comment]
hey @xahodo 

thanks for the suggestion! Just to mention #220, there will be more filtering than currently existing including the slideshow view from #224.

As a result, I think there will be an adaption to the start process, allowing the user to decide between unfiltered results, results matching hardware requirements and also some numeric limitation, as you suggested.

Also this might overlap with #217, but I will leave this open as your suggestion could be used as an alternative filtering mode.

**Comment by ajudacomunitaria** [comment]
Hi people.

*an idea*
![image](https://user-images.githubusercontent.com/125943841/220421024-7e765f42-5881-45aa-9393-e027ba5e8045.png)

*"source-code"*
```html
<label for="customRange1" class="form-label">Range results</label>
<input type="range" class="form-range" id="customRange1">
```

*"vs"*
![image](https://user-images.githubusercontent.com/125943841/220421575-094f529a-c5b1-4c03-9c28-fd6cb8d3c150.png)

**Comment by cmllr** [comment]
hey @eantho!

my feeling is, adding a arbitrary number limit leads to the issue that the use creates an artificial margin between the distribution result. This could lead to possible interesting results not displayed by the user if an number limit is enforced.

**Comment by villenmarx** [comment]
> my feeling is, adding a arbitrary number limit leads to the issue that the use creates an artificial margin between the distribution result. This could lead to possible interesting results not displayed by the user if an number limit is enforced.

Have you all ever thought of adopting a pagination interface with a limitation of 2 distros per page?

<img src="https://user-images.githubusercontent.com/126974943/222931033-4cd71c68-4996-4e69-81e5-07e5a06c9b05.png" alt="image" width="350px"/>

**Comment by cmllr** [comment]
> > my feeling is, adding a arbitrary number limit leads to the issue that the use creates an artificial margin between the distribution result. This could lead to possible interesting results not displayed by the user if an number limit is enforced.
> 
> Have you all ever thought of adopting a pagination interface with a limitation of 2 distros per page?
> 
> <img alt="image" width="450px" src="https://user-images.githubusercontent.com/126974943/223548904-e5810c5f-1717-463f-9f61-e77fd1db6ba9.png">

Not yet, but I think this could be an reasonable approach. Thanks for the remark!

**Comment by cliffbellman** [comment]
> Not yet, but I think this could be an reasonable approach. Thanks for the remark!

I believe that these two issues are very close, I would say equal. For example, 
- [Ask the user the amount of distributions to display as a result](https://github.com/distrochooser/distrochooser/issues/230)
- [Limit number of distro's shown in results by default](https://github.com/distrochooser/distrochooser/issues/150).

This could be done +- something like this:
<img src="https://user-images.githubusercontent.com/129481502/229040247-5f6ff62c-bbb7-448b-9f78-823f8538add2.png" alt="image" width="500"/>

**Comment by cmllr** [comment]
The result list will get updates for this using two changes: The slideshow view implemented with #224 and additionally, the default filter will be adapted to hide distributions with mostly negative or no-go-matches (implemented with #262). This allows the user to get a more compact match list, but if the user want's the entire list. This is now part of the beta proposal (#220) and will reach the distrochooser within the next large update.

Thanks everyone for the input!
